### ISSUE TITLE
Compatibility with Play 2.5

### ISSUE THREAD

**Issue by gwak** [hootsuite/akka-persistence-redis#14]
Hi,

Is akka-persistence-redis compatible with play 2.5. I've tried to migrate my persistent actors from levelDb to Redis-backed without success. I'm getting a runtime error :
```scale
[error] 13:00:36.804 a.a.ActorSystemImpl: Uncaught fatal error from thread [application-akka.persistence.dispatchers.default-plugin-dispatcher-11] shutting down ActorSystem [application]
java.lang.NoSuchMethodError: play.api.libs.functional.syntax.package$.functionalCanBuildApplicative(Lplay/api/libs/functional/Applicative;)Lplay/api/libs/functional/FunctionalCanBuild;
	at com.hootsuite.akka.persistence.redis.snapshot.SnapshotRecord$.<init>(SnapshotRecord.scala:34)
	at com.hootsuite.akka.persistence.redis.snapshot.SnapshotRecord$.<clinit>(SnapshotRecord.scala)
	at com.hootsuite.akka.persistence.redis.snapshot.RedisSnapshotStore.loadAsync(RedisSnapshotStore.scala:36)
	at akka.persistence.snapshot.SnapshotStore$$anonfun$1$$anonfun$applyOrElse$5.apply(SnapshotStore.scala:37)
	at akka.persistence.snapshot.SnapshotStore$$anonfun$1$$anonfun$applyOrElse$5.apply(SnapshotStore.scala:37)
	at akka.pattern.CircuitBreaker$State$class.materialize$1(CircuitBreaker.scala:298)
	at akka.pattern.CircuitBreaker$State$class.callThrough(CircuitBreaker.scala:315)
	at akka.pattern.CircuitBreaker$Closed$.callThrough(CircuitBreaker.scala:363)
	at akka.pattern.CircuitBreaker$Closed$.invoke(CircuitBreaker.scala:371)
	at akka.pattern.CircuitBreaker.withCircuitBreaker(CircuitBreaker.scala:115)
	at akka.persistence.snapshot.SnapshotStore$$anonfun$1.applyOrElse(SnapshotStore.scala:37)
	at scala.PartialFunction$OrElse.applyOrElse(PartialFunction.scala:170)
	at akka.actor.Actor$class.aroundReceive(Actor.scala:480)
	at com.hootsuite.akka.persistence.redis.snapshot.RedisSnapshotStore.aroundReceive(RedisSnapshotStore.scala:16)
	at akka.actor.ActorCell.receiveMessage(ActorCell.scala:526)
	at akka.actor.ActorCell.invoke(ActorCell.scala:495)
	at akka.dispatch.Mailbox.processMailbox(Mailbox.scala:257)
	at akka.dispatch.Mailbox.run(Mailbox.scala:224)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
```
Do you have any idea where the error might come from ?

Thanks

**Comment by ssong-van** [comment]
Hi, we haven't tried akka-persistence-redis with Play 2.5 yet. Let me try it out today and get back to you.

**Comment by gwak** [comment]
I've tried copying all classes into my project and updating dependencies to the latest, fixing some compilation errors and warnings and works, but I guess it's not ideal.

**Comment by ssong-van** [comment]
We just published 0.4.0 that pulls in Play 2.5. Please give it a try.

**Comment by gwak** [comment]
Hi, I've just tested 0.4.0 with Play 2.5 and it works like a charm! 
Thanks for reacting so quickly!
